apps/src/code-studio/components/progress/SelectedStudentInfo.jsx

```diff
@@ -0,0 +1,114 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import i18n from '@cdo/locale';
+import {TeacherPanelProgressBubble} from '@cdo/apps/code-studio/components/progress/TeacherPanelProgressBubble';
+import Button from '@cdo/apps/templates/Button';
+import {LevelStatus} from '@cdo/apps/util/sharedConstants';
+
+const styles = {
+  main: {
+    textAlign: 'center',
+    display: 'flex',
+    justifyContent: 'center',
+    flexDirection: 'column'
+  },
+  bubble: {
+    marginLeft: 81
+  }
+};
+
+export class SelectedStudentInfo extends React.Component {
+  static propTypes = {
+    selectedStudent: PropTypes.object,
+    level: PropTypes.object,
+    inMiniRubricExperiment: PropTypes.bool
+  };
+
+  onUnsubmit = () => {
+    $.ajax({
+      url: `/user_levels/${this.props.level.id}`,
+      method: 'PUT',
+      data: {
+        user_level: {
+          best_result: 1,
+          submitted: false
+        }
+      }
+    })
+      .done(data => {
+        // Let's just refresh so that the dots are correct, etc.
+        location.reload();
+      })
+      .fail(err => console.error(err));
+  };
+
+  onClearResponse = () => {
+    $.ajax({
+      url: `/user_levels/${this.props.level.id}`,
+      method: 'DELETE'
+    })
+      .done(data => {
```

Comment:
nit: indentation on lines 50-54

Reply:
This is the way prettier has me format it.

Reply:
So happy we have 🤖🤖🤖 to decide these things for us now! 😛